saritasa_invocations/alembic.py

```diff
@@ -2,17 +2,19 @@
 
 import invoke
 
-from . import printing, python
+from . import _config, printing, python
 
 
 @invoke.task
 def run(context: invoke.Context, command: str) -> None:
     """Execute alembic command."""
-    config = context.config.get("saritasa_invocations", {})
-    alembic_command = config.get("alembic_command", "-m alembic")
-    python.run_python(
+    config: _config.Config = context.config.get(
```

Comment:
Let's move
```python
    config: _config.Config = context.config.get(
        "saritasa_invocations",
        _config.Config(),
    )
```
to separate util.
This will reduce copy-paste and should improve typing